lua/conform/init.lua

```diff
@@ -3,6 +3,7 @@ local M = {}
 ---@class (exact) conform.FormatterInfo
 ---@field name string
 ---@field command string
+---@field path? string
```

Comment:
We don't really need to add the path here, do we? We can just run `exepath` from the info window where we need the full path

Reply:
Sorry made a mistake, I meant `formatter.path` instead of `type` but here you're answering my question.
Yeah, I think we don't as we use it only in the info panel, so it can be produced there.